content/blog/tidymodels-2022-q2/index.Rmd

```diff
@@ -0,0 +1,206 @@
+
+- [spatialsample](https://www.tidyverse.org/blog/2022/06/spatialsample-0-2-0/)
+- [recipes and its extension packages](https://www.tidyverse.org/blog/2022/05/recipes-update-05-20222/)
+
+Since [our last roundup post](https://www.tidyverse.org/blog/2022/04/tidymodels-2022-q1/), there have been CRAN releases of 25 tidymodels packages. You can install these updates from CRAN with:
+
+```{r, eval = FALSE}
+install.packages(c(
+  "rsample", "spatialsample", "parsnip", "baguette", "multilevelmod", "discrim",
+  "plsmod", "poissonreg", "rules", "recipes", "embed", "themis", "textrecipes",
+  "workflows", "workflowsets", "tune", "yardstick", "broom", "dials", "butcher",
+  "hardhat", "infer", "stacks", "tidyposterior", "tidypredict"
+))
+```
+
+- [baguette](https://baguette.tidymodels.org/news/index.html#baguette-100)
+- [broom](https://broom.tidymodels.org/news/index.html#broom-080)
+- [butcher](https://butcher.tidymodels.org/news/index.html#butcher-020)
+- [dials](https://dials.tidymodels.org/news/index.html#dials-100)
+- [discrim](https://discrim.tidymodels.org/news/index.html#discrim-100)
+- [embed](TODO)
+- [hardhat](https://hardhat.tidymodels.org/news/index.html#hardhat-110)
+- [infer](https://infer.tidymodels.org/news/index.html#infer-v102)
+- [modeldata](TODO)
+- [multilevelmod](https://multilevelmod.tidymodels.org/news/index.html#multilevelmod-100)
+- [parsnip](https://parsnip.tidymodels.org/news/index.html#parsnip-100)
+- [plsmod](TODO)
+- [poissonreg](https://poissonreg.tidymodels.org/news/index.html#poissonreg-100)
+- [recipes](TODO)
+- [rsample](https://rsample.tidymodels.org/news/index.html#rsample-100)
+- [rules](https://rules.tidymodels.org/news/index.html#rules-100)
+- [spatialsample](https://spatialsample.tidymodels.org/news/index.html#spatialsample-020)
+- [stacks](https://stacks.tidymodels.org/news/index.html#stacks-023)
+- [textrecipes](TODO)
+- [themis](TODO)
+- [tidyposterior](https://tidyposterior.tidymodels.org/news/index.html#tidyposterior-100)
+- [tidypredict](https://tidypredict.tidymodels.org/news/index.html#tidypredict-049)
+- [tune](TODO)
+- [workflows](TODO)
+- [workflowsets](TODO)
+- [yardstick](https://yardstick.tidymodels.org/news/index.html#yardstick-100)
+
+The `NEWS` files are linked here for each package; you'll notice that there are a lot! We know it may be bothersome to keep up with all these changes, so we want to draw your attention to our recent blog posts above and also highlight a few more useful updates in today's blog post.
+
+## case weights
```

Comment:
This section feels kinda short. The problem I ran into when trying to write more paragraphs, was that they all felt very similar to anything written in https://www.tidyverse.org/blog/2022/05/case-weights/. The user-facing syntax hasn't changed since this post and it felt weird "copy-pasting" it over.

Reply:
I think I'd suggest copying at least a little over so folks get the flavor.